pandas/_libs/parsers.pyx

```diff
@@ -96,6 +81,11 @@ try:
 except NameError:
     basestring = str
 
+cdef extern from "src/numpy_helper.h":
+    object sarr_from_data(cnp.dtype, int length, void* data)
+    void transfer_object_column(char *dst, char *src, size_t stride,
```

Comment:
@chris-b1, @gfyoung don't we have an issue to remove support for structured arrays entirely? is that already deprecated?

Reply:
if not let's create one (and deprecate in 0.22)